component/libfuse/libfuse2_handler.go

```diff
@@ -160,7 +161,20 @@ func (lf *Libfuse) initFuse() error {
 	if runtime.GOOS == "windows" && lf.networkShare && common.IsDriveLetter(lf.mountPath) {
 		// TODO: We can support any type of valid network share path so this path could
 		// be configurable for the config file. But this is a good default.
-		opts = append(opts, "--VolumePrefix=\\server\\share")
+
+		// by default bucketName will be blank
+		bucketName := ""
+		// Borrow bucket-name string from attribute cache
+		if config.IsSet("s3storage.bucket-name") {
```

Comment:
Could you do the same thing but for Azure here?

Reply:
Good point. You want both, depending on which one we're using, right? Does that mean we should check the components section of the config to see which cloud we're using?
This might be a bit tricky. I'm not sure if `config.IsSet("components.s3storage")` (or `azstorage`) will work...

Reply:
I'm afraid the config component section is the only way this can be checked. Doesn't the mount fail if that information is missing in the config anyway?

Reply:
There is code that writes a default components section when it is missing from the config. That being said, it's fine to do something that _usually_ works, and base it on the config.

Reply:
I was wrong that the components section is the only place. Just checking the s3storage or azstorage section of the config is good enough and the mound fails without a container or bucket specified in the config.